platforms/software/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/BuildInitPluginIntegrationTest.groovy

```diff
@@ -60,6 +61,18 @@ class BuildInitPluginIntegrationTest extends AbstractInitIntegrationSpec {
         ]
     }
 
+    @SuppressWarnings('GroovyAssignabilityCheck')
+    def "init can be run with arguments #args and built afterwards"() {
+        expect:
+        succeeds(args)
+        succeeds 'build'
+
+        where:
+        args << [
```

Comment:
❓ Why is this an iteration?  Did you intend to add more?  I'd suggest we either add more iterations here or collapse it into the test and change the name to be more descriptive.  It's a little awkward as it is now.